olive/passes/pytorch/qlora.py

```diff
@@ -267,6 +269,12 @@ def _run_for_config(
         new_model.model = None
         # remove the device map since we don't want "auto" device map
         new_model.hf_config.model_loading_args.device_map = None
+        # remove model_overwrites from model_attributes since new config will be created
+        if new_model.model_attributes:
+            for k in QLoRA.model_overwrites:
+                if k in new_model.model_attributes:
+                    del new_model.model_attributes[k]
```

Comment:
Call .pop(k, None) would be more clear?

Reply:
Thanks! this looks cleaner too. Updated.